server_src/runners/state.js

```diff
@@ -3473,26 +3479,7 @@ class Runner {
         });
       }
 
-      await Runner.getProfile(settings.printer.index);
-      await Runner.getSettings(settings.printer.index);
-      await Runner.getOctoPrintSystenInfo(settings.printer.index);
-      Runner.getOctoPrintSystenInfo(settings.printer.index);
-      Runner.getUpdates(settings.printer.index);
-      Runner.getPluginList(settings.printer.index);
```

Comment:
Mark this in your memory that if there are problems with settings we can add any of it back. I dont think we will have to, but just note.